conf/base/catalog.yml

```diff
@@ -24,4 +24,76 @@ oa_int_downstream_incoming_citations:
   path: "s3://alphafold-impact/data/02_intermediate/openalex/works/cited_by/downstream/${globals:get.work_id, undefined}/"
   credentials: s3_credentials
 
+# CRUNCHBASE
+# Raw
+_cb: &_cb
+  type: pandas.CSVDataset
+  credentials: nesta_s3_credentials
 
+cb_acquisitions_raw:
+  <<: *_cb
+  filepath: s3://discovery-iss/data/crunchbase/Crunchbase_20240109_000941/acquisitions.csv
```

Comment:
In regards to shortening the filepath names, you should be able to pass the parameter in catalog_globals.yml, something like 
`diss:
  crunchbase: s3://discovery-iss/data/crunchbase/Crunchbase_20240109_000941`
  
  Then, in catalog, you should be able to do:
  `
  ...
    filepath: ${diss.crunchbase}/acquisitions.csv`